### PR TITLE
Update protobuf to v3.7.1

### PR DESCRIPTION
**Pull request by andreamlin**

### CODE REVIEW

**Comment by andreamlin** [comment]
PTAL

**Comment by andreamlin** [comment]
PTAL
`bazel build //...` fails, but it also fails on the master branch.

File: repositories.bzl [hunk 1]
```diff
@@ -18,10 +18,12 @@ def com_google_protoc_java_resource_names_plugin_repositories(
...
+
+    http_archive(
         name = "com_google_protobuf",
-        remote = "https://github.com/google/protobuf.git",
-        tag = "v3.6.1",
+        urls = ["https://github.com/protocolbuffers/protobuf/archive/v%s.zip" % _protobuf_version],
+        strip_prefix = "protobuf-%s" % _protobuf_version
     )
 
 _DEFAULT_PY_BUILD_FILE = """
```
Comment (vam-google) [inline repositories.bzl]: Please add the following at the end of the `com_google_protobuf` macro:

```python
    http_archive(
        name = "bazel_skylib",
        sha256 = "bbccf674aa441c266df9894182d80de104cabd19be98be002f6d478aaa31574d",
        strip_prefix = "bazel-skylib-2169ae1c374aab4a09aa90e65efe1a3aad4e279b",
        urls = ["https://github.com/bazelbuild/bazel-skylib/archive/2169ae1c374aab4a09aa90e65efe1a3aad4e279b.tar.gz"],
    )

    http_archive(
        name = "net_zlib",
        build_file = "@com_google_protobuf//:third_party/zlib.BUILD",
        sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1",
        strip_prefix = "zlib-1.2.11",
        urls = ["https://zlib.net/zlib-1.2.11.tar.gz"]
    )
```

These are the transitive deps of `protobuf-3.7.1`. Protobuf does not expose `repositories.bzl` properly yet, so we have to duplicate it directly here.

Comment (andreamlin) [reply repositories.bzl]: Oh right! I also had to add a `native.bind()` for zlib to net_zlib like in gapic-generator for this to work.

**Comment by andreamlin** [comment]
`bazel build //...` is happy now!

PTAL

**Pull request merged**